modules/integration/src/test/scala/scala/cli/integration/CompileTestDefinitions.scala

```diff
@@ -101,7 +112,7 @@ abstract class CompileTestDefinitions(val scalaVersionOpt: Option[String])
             |    println(message)
             |}
             |""".stripMargin,
-        os.rel / "Tests.scala" ->
+        os.rel / "Tests.test.scala" ->
```

Comment:
Same as above, do we really need that as we already have `using target.scope "test"`?

Reply:
Yes, we don't need change name of file to `*.test.scala` if we have `using target.scope "test"`